build.sh

```diff
@@ -243,12 +246,18 @@ isMSBuildOnNETCoreSupported()
                 "fedora.24-x64")
                     __isMSBuildOnNETCoreSupported=1
                     ;;
+                "fedora.27-x64")
+                    __isMSBuildOnNETCoreSupported=1
+                    ;;
                 "opensuse.13.2-x64")
                     __isMSBuildOnNETCoreSupported=1
                     ;;
                 "opensuse.42.1-x64")
                     __isMSBuildOnNETCoreSupported=1
                     ;;
+                "opensuse.42.3-x64")
+                    __isMSBuildOnNETCoreSupported=1
+                    ;;
```

Comment:
@janvorli you probably already know this, but be sure to submit a PR adding ubuntu1804 to this whenever the bootstrap cli is available.

Reply:
I know :-)